src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1308,7 +1308,7 @@ private void OnControlStateChangedForShadow(object obj, ControlStateChangedEvent
                     return;
                 }
 
-                Tizen.NUI.Object.SetProperty(swigCPtr, View.Property.SHADOW, boxShadow == null ? new PropertyValue() : boxShadow.ToPropertyValue(this));
+                Tizen.NUI.Object.SetProperty(swigCPtr, View.Property.SHADOW, boxShadow == null ? new PropertyValue() : boxShadow?.ToPropertyValue(this));
```

Comment:
> We need Null-conditional operator(`?.`) here?
> 
> Because of null check condition expression(`boxShadow == null`), `imageshadow` is always non-Null.

OK. I will revert it